src/static/chemiscope.css

```diff
@@ -37,14 +37,120 @@
     z-index: 10;
 }
 
-.chsp-open-viewer-settings {
+.chsp-open-viewer-settings{
     position: absolute;
     top: 5px;
     right: 5px;
     /* Place on top of jsmol applet */
     z-index: 10;
 }
 
+.chsp-circle-button {
+  border-radius: 100%;
+  border: solid thin black;
+  position: absolute;
+  top: 10px;
+  height: 20px;
+  width: 20px;
+  /* Place on top of jsmol applet */
+  z-index: 10;
+}
+
+.vertical-rect {
+  height:20px;
+  width:10px
+}
+.horizontal-rect {
+  height:10px;
+  width:20px
+}
+
+.chsp-active {
```

Comment:
`.chsp-active` and `.chsp-inactive` need a bit more context in their name. They are used with the viewer, right?